lucet-runtime/lucet-runtime-tests/src/timeout.rs

```diff
@@ -281,32 +330,41 @@ macro_rules! timeout_tests {
                 .new_instance(module)
                 .expect("instance can be created");
 
+            // Spawn a thread to terminate the instance after waiting for 100ms.
             let kill_switch = inst.kill_switch();
+            let helper_1 = thread::Builder::new()
+                .name("helper-1".to_owned())
```

Comment:
```suggestion
                .name("killswitch-1".to_owned())
```
(and the same for `helper-2`)? Mostly so the `expect` below are clearer about what did/didn't panic.